Modules/UtilsKit/Sources/Extensions/Color+Extension.swift

```diff
@@ -0,0 +1,20 @@
+// Leka - iOS Monorepo
+// Copyright APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+import SwiftUI
+
+// swiftlint:disable identifier_name
+public extension Color {
+    init(hex: UInt, alpha: Double = 1) {
+        self.init(
+            .sRGB,
```

Comment:
any reason you're using `.sRGB` instead of `.displayP3`?

if not let's use `.dislayP3`